app/src/main/java/com/framgia/forder/data/model/CartItem.java

```diff
@@ -0,0 +1,152 @@
+package com.framgia.forder.data.model;
+
+import com.google.gson.annotations.Expose;
+import com.google.gson.annotations.SerializedName;
+
+import static com.framgia.forder.utils.Constant.FORMAT_PRICE;
+import static com.framgia.forder.utils.Constant.UNIT_MONEY;
+
+/**
+ * Created by tri on 20/04/2017.
+ */
+
+public class CartItem {
+
+    @SerializedName("status")
+    @Expose
+    private int mStatus;
+    @SerializedName("product_id")
+    @Expose
+    private int mProductId;
+    @SerializedName("user_id")
+    @Expose
+    private int mUserId;
+    @SerializedName("quantity")
+    @Expose
+    private int mQuantity;
+    private int mDomainId;
+    private int mShopId;
+    private String mProductName;
+    private String mProductImage;
+    private double mPrice;
+    private String mStartHour;
+    private String mEndHour;
+    private Double mTotal;
+
+    public CartItem() {
+    }
+
+    public CartItem(int domainId, int shopId, int quantity, int productId, String productName,
+            String productImage, double price, String startHour, String endHour, Double total) {
+        mDomainId = domainId;
+        mShopId = shopId;
+        mQuantity = quantity;
+        mProductId = productId;
+        mProductName = productName;
+        mProductImage = productImage;
+        mPrice = price;
+        mStartHour = startHour;
+        mEndHour = endHour;
+        mTotal = total;
+    }
+
+    public int getDomainId() {
+        return mDomainId;
+    }
+
+    public void setDomainId(int domainId) {
+        mDomainId = domainId;
+    }
+
+    public int getShopId() {
+        return mShopId;
+    }
+
+    public void setShopId(int shopId) {
+        mShopId = shopId;
+    }
+
+    public int getQuantity() {
+        return mQuantity;
+    }
+
+    public void setQuantity(int quantity) {
+        mQuantity = quantity;
+    }
+
+    public int getProductId() {
+        return mProductId;
+    }
+
+    public void setProductId(int productId) {
+        mProductId = productId;
+    }
+
+    public String getProductName() {
+        return mProductName;
+    }
+
+    public void setProductName(String productName) {
+        mProductName = productName;
+    }
+
+    public String getProductImage() {
+        return mProductImage;
+    }
+
+    public void setProductImage(String productImage) {
+        mProductImage = productImage;
+    }
+
+    public double getPrice() {
+        return mPrice;
+    }
+
+    public void setPrice(double price) {
+        mPrice = price;
+    }
+
+    public String getStartHour() {
+        return mStartHour;
+    }
+
+    public void setStartHour(String startHour) {
+        mStartHour = startHour;
+    }
+
+    public String getEndHour() {
+        return mEndHour;
+    }
+
+    public void setEndHour(String endHour) {
+        mEndHour = endHour;
+    }
+
+    public String getFormatPrice() {
+        return String.format(FORMAT_PRICE, mTotal) + UNIT_MONEY;
```

Comment:
WARNING: [DefaultLocale] Implicitly using the default locale is a common source of bugs: Use `String.format(Locale, ...)` instead
 Calling `String#toLowerCase()` or `#toUpperCase()` *without specifying an explicit locale* is a common source of bugs. The reason for that is that those methods will use the current locale on the user's device, and even though the code appears to work correctly when you are developing the app, it will fail in some locales. For example, in the Turkish locale, the uppercase replacement for `i` is *not* `I`.

If you want the methods to just perform ASCII replacement, for example to convert an enum name, call `String#toUpperCase(Locale.US)` instead. If you really want to use the current locale, call `String#toUpperCase(Locale.getDefault())` instead.